addons/upgrade/to-13.0-convert-h3cswitch-types.pl

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/perl
```

Comment:
Please rename the file from addons/upgrade/to-13.0-convert-h3cswitch-types.pl to addons/upgrade/to-13.1-convert-h3cswitch-types.pl